debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java

```diff
@@ -1132,6 +1132,10 @@ protected Object handleUnknownData(Column column, Field fieldDefn, Object data)
         if (data == UnchangedToastedReplicationMessageColumn.UNCHANGED_TOAST_VALUE) {
             return toastPlaceholderString;
         }
+        if ((data instanceof List) && (((List) data).size() > 0) &&
```

Comment:
Could you please just add a comment line above explaining what's going on for future maintainers? Code is fine.